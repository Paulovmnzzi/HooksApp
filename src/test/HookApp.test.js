const { shallow } = require("enzyme")
import HooksApp from '../HooksApp'

describe('Hook app', () => {
    test('Hook app Snapshot', () => {
        
        const wrapper = shallow(<HooksApp />)
        expect(wrapper).toMatchSnapshot();
        


    })
    

})
