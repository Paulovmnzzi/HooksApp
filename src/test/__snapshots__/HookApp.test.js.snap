// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hook app Hook app Snapshot 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
